src/EpicRoadmap/react/Components/EpicRoadmapView.tsx

```diff
@@ -54,6 +54,14 @@ class EpicRoadmapViewContent extends React.Component<IEpicRoadmapViewProps, IEpi
             return (
                 <Spinner size={SpinnerSize.large} className="loading-indicator" label="Loading..." />
             );
+        } else if (uiState === UIStatus.Error) {
+            return (
+                <MessageBar
+                    messageBarType={MessageBarType.error}
+                    isMultiline={false}
+                >{"Epic Roadmap is not supported for Basic process."}
```

Comment:
Please update the message to say that Epic Roadmap requires atleast two levels of Portfolio backlogs, and also link to the page which describes portfolio backlogs.

Reply:
As discussed offline, I am not including the link as of now as there is no doc available describing portfolio backlogs correctly.